Matrix/SearchMatrix.py

```
@@ -0,0 +1,76 @@
+          right=len(matrix)-1
+          while left<=right:
+                mid=(left+right)//2 
+                if result[mid]==target:
+                    return True 
+                elif result[mid]<target:
+                     left=mid+1
+                else:
+                    right=mid-1
+          return False 
+# Apply Binary Search 
+class Solution:
```

Ruff (0.11.9)

Redefinition of unused `Solution` from line 41

(F811)